tests/TestOkWin32GdiPointPatch.py

```
@@ -0,0 +1,77 @@
+import ctypes.wintypes as wintypes
+import unittest
+
+from ok.ui.overlay import win32_gdi
+from src.patches.win32_gdi_point_patch import install_win32_gdi_point_patch
+
+
+class TestOkWin32GdiPointPatch(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        # 记录补丁前的自定义 POINT 类（补丁会把它替换为 wintypes.POINT）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)